ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -264,7 +264,7 @@ public void run() {
    * @throws ReplicationException
    * @throws InterruptedException
    */
-  protected List<ExchangeMetadataResponse> exchangeMetadata(ConnectedChannel connectedChannel,
```

Comment:
It is used only inside the package. I'm not sure why it was `protected` before (no inheritors outside the package since `ReplicaThread` itself is package private) but this is the right visibility now.